pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Description
+
+This PR closes #${issue_number}.
+
+<!-- Write a description to explain changes that are not described in the initatial issues.
+What were the reasons for those changes?
+Which decisions did you make and why? 
+What else should reviewers know about your changes? -->
+
+## Testing
+
+<!-- How did you test your changes?
+Which new tests did you add?
+Did you run anything manually? -->
+
+## Checklist
+
+- [ ] Code is regenerated if needed (`task gen`).
+- [ ] Code is formatted (`task fmt`).
+- [ ] Docs are formatted (`task docs-fmt`).
+- [ ] Tests passed (`task test`).
+- [ ] Linters passed (`task lint`).
+- [ ] PR assignee is set.
+- [ ] PR labels are set.
+- [ ] Project and milestone are set.
+- [ ] Critical codecov complaints are considered.
+- [ ] PR title and body are in accordance with the Conform PR GitHub Action.
+- [ ] Other GitHub Actions passed.
```

Comment:
I would appreciate it if there is something that you assume is obvious and it's not on the list.